src/Microsoft.AspNet.Mvc.Core/Rendering/Html/HtmlHelperOfT.cs

```diff
@@ -70,10 +74,11 @@ public override void Contextualize([NotNull] ViewContext viewContext)
         }
 
         /// <inheritdoc />
-        public HtmlString DisplayFor<TValue>([NotNull] Expression<Func<TModel, TValue>> expression,
-                                             string templateName,
-                                             string htmlFieldName,
-                                             object additionalViewData)
+        public HtmlString DisplayFor<TResult>(
+            [NotNull] Expression<Func<TModel, TResult>> expression,
+            string templateName,
+            string htmlFieldName,
```

Comment:
lol man I'm really confused now, is this supposed to be an actual `htmlFieldName`?